easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.5-intel-2015a.eb

```diff
@@ -0,0 +1,23 @@
+name = 'GROMACS'
+version = '4.6.5'
```

Comment:
Ok you are right, 4.6.7 should work too for g_mmpbsa. I can have a look and add the easyconfigs for Gromacs 4.6.7 and g_mmpbsa too at some point later?